benchmark/benchmark/seed.py

```diff
@@ -34,24 +32,22 @@ def _kill_nodes(self):
         except subprocess.SubprocessError as e:
             raise BenchError('Failed to kill testbed', e)
 
-    def run(self, committee_path):
-        assert isinstance(committee_path, Path)
+    def run(self, starting_data_port):
```

Comment:
We probably want to sanitize `starting_data_port` as an `int`, since this will allow us to detect passing the wrong argument without a long-running command launching.